cypress_shared/pages/temporary-accommodation/manage/bedspaceSearch.ts

```diff
@@ -67,15 +67,15 @@ export default class BedspaceSearchPage extends Page {
     this.getLegend('Property attributes')
     this.getLegend('Occupancy (optional)')
     searchParameters.attributes
-      .filter(attribute => attribute === 'wheelchairAccessible')
+      .filter(attribute => attribute !== 'isWheelchairAccessible')
       .forEach(attribute => {
         this.checkCheckboxByNameAndValue('attributes[]', attribute)
       })
 
     this.getLegend('Bedspace attributes')
     this.getLegend('Accessibility (optional)')
     searchParameters.attributes
-      .filter(attribute => attribute !== 'wheelchairAccessible')
+      .filter(attribute => attribute === 'isWheelchairAccessible')
```

Comment:
I discovered a small issue here with the logic where it was only returning single/shared value when it should be returning wheelchair accessible attributes